application/tests/DbTestCase.php

```diff
@@ -2,4 +2,12 @@
 
 class DbTestCase extends CIPHPUnitTestDbTestCase
 {
+    public static function setUpBeforeClass(): void
```

Comment:
Normally, my preference is to reset the data between each test rather than between each test class, so that there's no leakage between tests, but it seems like setting up such a large database is quite expensive. (On my humble machine, this takes between 30 to 40s to do the reload.)

I think there's a few options we could explore for speeding this up:
* Use a reduced-size data set for the tests - Less production-like, but fast and you get some data set up for free
* Use a schema-only data set for the tests - Even _less_ production like, but you get full control of the data being set up and it should reload really fast
* Maybe: Benchmark the `DatabaseSeeder` and see if we can get that to run faster somehow, though it's probably just the data size making it slow